test/CompressionPlugin.test.js

```diff
@@ -0,0 +1,32 @@
+import Plugin from '../src/index';
+import { cleanErrorStack, createCompiler, compile } from './helpers';
```

Comment:
Ideally we agree to use `@webpacl-contrib/test-utils` to keep the cpmmon helpers in one place in the future, this can be done later as e.g the `cleanErrorStack` helper is currently missing there

Reply:
@michael-ciniawsky `@webpacl-contrib/test-utils` is good for testing loader, but many plugin require own environment, `test-utils` is vacuum implementation that does not fit in most cases